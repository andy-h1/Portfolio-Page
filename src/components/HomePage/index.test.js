import React from 'react';
import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import { HomePage } from '.';

describe('Home Page Component', () => {
  it('shoulder render', () => {
    const { container } = render(
      <MemoryRouter>
        <HomePage />
      </MemoryRouter>
    );
    expect(container.firstChild).toMatchSnapshot();
  });
});
