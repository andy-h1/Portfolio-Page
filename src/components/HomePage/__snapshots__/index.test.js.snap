// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page Component shoulder render 1`] = `
<div
  class="sc-fznKkj iDLDLE"
>
  <div
    class="sc-fzplWN gOAnAS"
  >
    <img
      alt="andy"
      class="sc-fznyAO dSDlBX"
      src="profile.jpg"
    />
    <p
      class="sc-fznZeY edopax"
    >
      Hey! Welcome to my website that I've built in React. Currently I am working as a tech recruiter at Deliveroo but hoping to create enough React projects to make the move over to becoming a dev.
    </p>
    <p
      class="sc-fznZeY edopax"
    >
      I've had a very fruitful career so far, coming from a nursing background to becoming a recruiter and now looking to make my final move over to development where my interests lies. My end goal is to build something useful for the healthcare sector but in the meantime I'm honing my skills in everything front-end.
    </p>
  </div>
  <div
    class="sc-AxjAm bfLLc"
  >
    <h2
      class="sc-AxirZ dDfKVJ"
    >
      Projects
    </h2>
    <a
      class="sc-AxiKw cCebkB"
      href="/weather"
    >
      <div
        class="sc-AxhCb cVoiXz"
      >
        <h3
          class="sc-AxhUy iAIcNb"
        >
          Weather App
        </h3>
        <p
          class="sc-AxgMl eOWHjm"
        >
          I built this weather app using OpenWeatherMap API to get the current weather data in your location. It has been built with React using styled-components.
        </p>
        <div
          class="sc-AxheI fpnsVe"
        >
          <ul
            class="sc-fzoLsD ihFyfP"
          >
            <li
              class="sc-fzpans eoBJOd"
            >
              react
            </li>
            <li
              class="sc-fzpans eoBJOd"
            >
              styled-components
            </li>
            <li
              class="sc-fzpans eoBJOd"
            >
              jest
            </li>
            <li
              class="sc-fzpans eoBJOd"
            >
              react-testing-library
            </li>
          </ul>
        </div>
        <span
          class="sc-Axmtr hAhsOL"
        >
          <img
            alt="cloudy"
            class="sc-AxmLO VXOhf"
            src="lightCloudsDay.png"
          />
        </span>
      </div>
    </a>
    <a
      class="sc-AxiKw cCebkB"
      href="/Form"
    >
      <div
        class="sc-AxhCb cVoiXz"
      >
        <h3
          class="sc-AxhUy iAIcNb"
        >
          Sign-Up Form
        </h3>
        <p
          class="sc-AxgMl eOWHjm"
        >
          I built this demo sign-up form, hooking the data entries to Firebase's cloud-based database - Firestore. It has been built with React, Firebase and styled-components.
        </p>
        <div
          class="sc-AxheI fpnsVe"
        >
          <ul
            class="sc-fzoLsD ihFyfP"
          >
            <li
              class="sc-fzpans eoBJOd"
            >
              react
            </li>
            <li
              class="sc-fzpans eoBJOd"
            >
              styled-components
            </li>
            <li
              class="sc-fzpans eoBJOd"
            >
              jest
            </li>
            <li
              class="sc-fzpans eoBJOd"
            >
              react-testing-library
            </li>
          </ul>
        </div>
        <span
          class="sc-Axmtr hAhsOL"
        >
          <img
            alt="form"
            class="sc-fzozJi dlipve"
            src="formImage.png"
          />
        </span>
      </div>
    </a>
  </div>
</div>
`;
